### PR TITLE
get_detail

### PR DESCRIPTION
**Pull request by anhhuy1010**

### CODE REVIEW

File: controllers/cart.go [hunk 1]
```diff
@@ -52,14 +53,15 @@ func (cartCtr CartController) Create(c *gin.Context) {
 	}
 	cartData := &models.Carts{}
 	if req.CartUuid == "" {
-		cartData.Uuid = util.GenerateUUID()
+		cartData.CartUuid = util.GenerateUUID()
```
Comment (namleDiqit) [inline controllers/cart.go]: Why change Uuid to CartUuid ? It is unnecessary

File: controllers/cart.go [hunk 2]
```diff
@@ -52,14 +53,15 @@ func (cartCtr CartController) Create(c *gin.Context) {
...
 		cartData.Total = 0
+
 		if _, err := cartData.Insert(); err != nil {
 			c.JSON(http.StatusInternalServerError, respond.UpdatedFail())
 			return
 		}
 	} else {
-		cond := bson.M{"uuid": req.CartUuid}
+		cond := bson.M{"cart_uuid": req.CartUuid}
```
Comment (namleDiqit) [inline controllers/cart.go]: same above

File: controllers/cart.go [hunk 3]
```diff
@@ -73,13 +75,13 @@ func (cartCtr CartController) Create(c *gin.Context) {
...
 	cartItem := models.CartItem{
-		Uuid:         util.GenerateUUID(),
-		CartUuid:     cartData.Uuid,
+		CartUuid:     cartData.CartUuid,
 		ProductUuid:  req.ProductUuid,
 		ProductName:  productDetail.Name,
 		Quantity:     1,
 		ProductPrice: productDetail.Price,
 		ProductTotal: 1 * productDetail.Price,
+		CartItemUuid: util.GenerateUUID(),
```
Comment (namleDiqit) [inline controllers/cart.go]: Why change Uuid to CartItemUuid in collection cartItem ?

File: controllers/cart.go [hunk 4]
```diff
@@ -89,3 +91,73 @@ func (cartCtr CartController) Create(c *gin.Context) {
 
 	c.JSON(http.StatusOK, respond.Success(cartItem, "Update successfully"))
 }
+func (cartCtl CartController) Detail(c *gin.Context) {
+	cartItemModel := new(models.CartItem)
```
Comment (namleDiqit) [inline controllers/cart.go:95]: missing logic check cart not found

**Review by LeHNam — APPROVED** [review]

**Pull request merged**